### ISSUE TITLE
Download_AWS_INMET_daily error in column names

### ISSUE THREAD

**Issue by ammaciel** [FilgueirasR/BrazilMet#4]
@FilgueirasR, I'm getting this error. I'm attempt with the stations A001, A042 and A711. 

```
df <- download_AWS_INMET_daily(station = "A042", start_date = "2022-01-01", end_date = "2022-12-31")
trying URL 'https://portal.inmet.gov.br/uploads/dadoshistoricos/2022.zip'
Content type 'application/zip' length 90362801 bytes (86.2 MB)
==================================================
downloaded 86.2 MB

Error in make.names(col.names, unique = TRUE) : 
  invalid multibyte string 3
```

Could you help?
This package is very useful!

**Comment by FilgueirasR** [comment]
Hi @ammaciel !
Sure, I will do anything to help you. First, install the package using the version of github.

To install the last version of BrazilMet  package follow this steps:

devtools::install_github("FilgueirasR/BrazilMet")

**Comment by ammaciel** [comment]
Thank you for the answer. This resolved the error.

The package is very interesting. I checked that it only returns [automatic stations](https://portal.inmet.gov.br/paginas/catalogoaut), is there a plan to add [conventional stations](https://portal.inmet.gov.br/paginas/catalogoman) as well? And is there also a possibility to add the name and state of the stations? In some works that I do, only the station name is passed, without its code.

**Comment by FilgueirasR** [comment]
Thank you for the feed Back @ammaciel ! 

We do not have in mind, at least now, the insertion of data from conventional stations, we can think about it. Regarding inputting the station name and state, we can input it shortly for you.

**Comment by FilgueirasR** [comment]
Hi @ammaciel ! 
I already added the name and state of the stations as you suggested. 
Hope you enjoy!

**Comment by ammaciel** [comment]
Hi @FilgueirasR!

Thanks for considering my suggestion. I very much appreciated your solution. 
I tested this new version from github master branch, and I verify that you added the name and state of the stations in download_AWS_INMET_daily function. It's excellent! 

However, if I desire to download station(s) data based on its name or to a state (UF)  applying a for loop, for instance, I still need to know the code of the station previously. In this case, I think it will be more interesting if the name and state stations were also added in the see_stations_info function return. Thus, I will run the function download_AWS_INMET_daily only for desired stations for Paraná state, for instance, or the one that I know the station name and after running the see_stations_info function, I found its code. 

But, it is only a suggestion to improve your package!
